src/bundle/Resources/views/themes/admin/ui/search/form.html.twig

```diff
@@ -19,7 +22,75 @@
             <span class="ez-btn ez-btn--filter-label">{{ 'search.filters'|trans|desc('Filters') }}</span></button>
     </span>
 </div>
-<div class="ez-filters mt-4 ml-4 pt-2 {% if not filters_expanded %} ez-filters--collapsed{% endif %}">
+
+<div class="ez-search-criteria-tags ml-4">
+    {% if form.content_types.vars.data is not empty %}
+        {% for contentType in form.content_types.vars.data %}
+            {{ include('@ezdesign/ui/search_tag.html.twig', {
+                'content': contentType.name,
+                'title': "#{'search.content.type'|trans|desc('Content Type:')} #{contentType.name}",
+                'targetSelector': "#search_content_types_#{contentType.identifier}",
+                'btnClass': "ez-tag__remove-btn--content-types",
+            }) }}
+        {% endfor %}
+    {% endif %}
+
+    {% if form.section.vars.data is not empty %}
+        {{ include('@ezdesign/ui/search_tag.html.twig', {
+            'content': form.section.vars.data.name,
+            'title': "#{'search.section'|trans|desc('Section:')} #{form.section.vars.data.name}",
+            'btnClass': "ez-tag__remove-btn--section",
+        }) }} 
+    {% endif %}
+
+    {% if form.subtree.vars.value is not empty %}
+        {% if subtree_selected %}
+            {% set path_locations = ez_path_to_locations(form.subtree.vars.value) %}
+            {% for location in path_locations %}
+                {% set content_breadcrumbs = content_breadcrumbs ~ ez_content_name(location.contentInfo) %}
+                {% if not loop.last %}
+                    {% set content_breadcrumbs = content_breadcrumbs ~ ' / ' %}
+                {% endif %}
+            {% endfor %}
+        {% endif %}
+
+        {{ include('@ezdesign/ui/search_tag.html.twig', {
+            'content': content_breadcrumbs,
+            'title':  "#{'search.subtree'|trans|desc('Subtree:')} #{content_breadcrumbs}",
+            'btnClass': "ez-tag__remove-btn--subtree",
+        }) }}
+    {% endif %}
+
+    {% if form.vars.value.lastModified is not empty %}
+        {% set startDate = form.vars.value.lastModified.start_date|ez_short_date %}
```

Comment:
Correct me if I'm wrong but according to convention identifiers should use underscore as a word separator e.g. https://github.com/ezsystems/ezplatform-admin-ui/pull/1265/files#diff-4f6a06e0b494e350331247a4b1dd2c54R48